.github/workflows/bun-types.yml

```
@@ -0,0 +1,41 @@
+env:
+  BUN_VERSION: "canary"
+
+jobs:
+  typecheck:
+    name: "TypeScript types"
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Bun
+        uses: ./.github/actions/setup-bun
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/setup-bun": line 5: unexpected key "type" for definition of input "download-url"

(action)